guides/user/pipeline/searching/index.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: single
+title:  "Searching for Triggered Pipeline Executions"
+sidebar:
+  nav: guides
+redirect_from: /guides/user/triggers/searching
+---
+
+{% include toc %}
+
+There exists an API that can be used to search for pipeline executions given information about what triggered them. This guide details the best practices to structure your triggers so that their corresponding pipeline executions can be easily queried.
```

Comment:
"This guide describes how to use the _____ API to search for pipeline executions triggered by Pub/Sub messages or webhooks. "